.gitlab-ci.yml

```diff
@@ -29,7 +29,7 @@ stages:
 
 variables:
   # Tested CMake versions
-  CMAKE_MINIMUM_VERSION: 3.5.2
+  CMAKE_MINIMUM_VERSION: 3.10.2
   CMAKE_LATEST_VERSION: 3.17.0
```

Comment:
We may be able to update to 3.18

Reply:
CI passed.  Are we going to try for cmake 3.18 or just merge this commit in as is?  ROCm 4.2 feature complete date is Friday.

Reply:
I will just merge this in for now.  We can try for cmake 3.18 for 4.3